scrapi/linter/document.py

```diff
@@ -129,7 +129,7 @@ class NormalizedDocument(BaseDocument):
 
     def __repr__(self):
         return "NormalizedDocument(source='{}', url='{}', providerUpdatedDateTime='{}', ...)".format(
-            self.attributes['shareProperties']['source'],
-            self.attributes['uris']['canonicalUri'],
-            self.attributes['providerUpdatedDateTime']
+            self.attributes.get('shareProperties', {}).get('source'),
+            self.attributes.get('uris', {}).get('canonicalUri'),
+            self.attributes.get('providerUpdatedDateTime')
```

Comment:
yeah, but if it isn't this will throw an error, which requires the ```__repr__``` of the object, which will throw an exception at ```__repr__()``` instead of the original error.